libdevcore/Base64.cpp

```diff
@@ -36,7 +36,11 @@ static inline bool is_base64(byte c)
 	return (isalnum(c) || (c == '+') || (c == '/'));
 }
 
+#ifdef __MIC__
+byte find_base64_char_index(byte c)
```

Comment:
I had to remove both "static" and "inline" for Intel Compiler with -mmic.
Will this modification properly function with other build systems?